app/vmagent/remotewrite/client.go

```diff
@@ -295,6 +295,16 @@ again:
 	}
 	metrics.GetOrCreateCounter(fmt.Sprintf(`vmagent_remotewrite_requests_total{url=%q, status_code="%d"}`, c.sanitizedURL, statusCode)).Inc()
 	if statusCode == 409 || statusCode == 400 {
+		body, err := ioutil.ReadAll(resp.Body)
+		_ = resp.Body.Close()
+		if err != nil {
+			logger.Errorf("sending a block with size %d bytes to %q was rejected (skipping the block): status code %d; "+
+				"failed to read response body: %s",
+				len(block), c.sanitizedURL, statusCode, err)
+		} else {
+			logger.Errorf("sending a block with size %d bytes to %q was rejected (skipping the block): status code %d; response body: %s",
```

Comment:
This error message must be throttled (e.g. to log it once per 5-10 seconds), otherwise it may flood the error log when vmagent processes high amounts of metrics.

Reply:
Please see https://github.com/VictoriaMetrics/VictoriaMetrics/pull/1956/commits/71a516b411000f94a6595686571e161b95bf85eb